apps/platform-app/src/app/[locale]/(profile)/(routes)/p/[username]/page.tsx

```diff
@@ -14,7 +18,9 @@ export async function generateMetadata({
   params: { username: string }
 }) {
   try {
-    const selectedProfile = await getProfileByGithubUsername(params.username)
+    const selectedProfile =
+      (await getProfileByGithubUsername(params.username)) ??
```

Comment:
**nit:** We should go IMO with `findProfileByUsername` without specifying by which provider. I will leave more details below